pyiron_base/job/jobtype.py

```diff
@@ -66,7 +68,7 @@ def __new__(cls, class_name, project, job_name, job_class_dict, delete_existing_
                 "No HDF5 file found - remove database entry and create new job! {}".format(job.job_name)
             )
             delete_existing_job = True
-        if delete_existing_job:
+        if delete_existing_job or job.status.aborted and delete_aborted_job:
```

Comment:
```suggestion
        if delete_existing_job or (job.status.aborted and delete_aborted_job):
```

It actually runs fine how you had it but I feel like the brackets better connect the single clause.